inst/tests/tests.Rraw

```diff
@@ -6312,15 +6312,13 @@ options(datatable.optimize = Inf)
 
 # fread dec=',' e.g. France
 test(1439, fread("A;B\n1;2,34\n", dec="12"), error="nchar(dec) == 1L is not TRUE")
-test(1440, fread("A;B\n8;2,34\n", dec="1"), data.table(A=8L, B="2,34"))
```

Comment:
`!dec %chin% c('.', ',')` is implied incorrect by L1179 of `fread.c` @ master:

> dec='' not allowed. Should be '.' or ','

`?fread` is not as strict:

> The decimal separator as in utils::read.csv. If not "." (default) then usually ",". See details.

Took a conservative approach for now, and blocked `!dec %chin% c(',', '.')`. But that broke two tests (`1440` and `1444.2` here).

`dec='1'` and `dec='*'` ran without errors... happy to restore this to work, and change the error/message found in `fread.c`.